src/styles/bank_transactions.scss

```diff
@@ -417,7 +418,7 @@
   }
 
   &.Layer__bank-transaction-row--removing {
-    display: none;
+    opacity: 0.99;
```

Comment:
Why are we using 0.99 opacity?

Reply:
Good point! we don't need this any longer. Previously, we used change on `opacity` or `top` to know when animation ends and remove permanently transaction from the list. Now, we use different approach.